TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/models/command_runner.py

```diff
@@ -27,10 +27,10 @@ def __init__(self, command_root: str, command_args: Dict[str, Any], sensitive_fi
         self.run_as_detached = run_as_detatched
         self.log_file = log_file
 
-    def run(self) -> CommandResult:
+    def run(self, print_to_console=True) -> CommandResult:
```

Comment:
This changes the default behavior, right? I'm fine with the option, but slightly skeptical that we want to change the default for every command we run. Can you justify that/show an example?

Reply:
From my testing and this previous comment `# Pass None to stdout and stderr to not capture output and show in terminal` any command that this would run would automatically print its output to the console so I tried to leave that as the default. In the context of the kubectl commands I normally just want to parse the output and not show it. The metadata migrate command for instance uses this runner to show its output from executing its command

Reply:
Oh, I see, I think I had what that `None` did backwards. 👍